src/Raven.Studio/typescript/viewmodels/database/indexes/indexErrors.ts

```diff
@@ -289,6 +319,20 @@ class indexErrors extends viewModelBase {
             this.gridController().reset();
         }
     }
+
+    clearIndexErrors() {
+        const indexesToClear = this.selectedIndexNames();
+        
+        if (indexesToClear.length > 0) {
```

Comment:
no need to check that - button is not enabled?

Reply:
right - done